src/keystore.h

```diff
@@ -125,6 +125,13 @@ USE_RESULT bool keystore_bip39_mnemonic_to_seed(
     uint8_t* seed_out,
     size_t* seed_len_out);
 
+/**
+ * Fills a uint8_t buffer with a fingerprint,
```

Comment:
"a fingerprint" is a bit vague, what about "with the fingerprint of the root xpub as specified in https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki#serialization-format"?

Reply:
Done.